style.css

```diff
@@ -52,6 +52,45 @@
     display: none;
   }
 
+  /* menu toggle styling */
+  .navbar-show {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 100%;
+    background: #6070ff;
+    background-blend-mode: multiply;
+    margin: 0;
+    padding: 0;
+    opacity: 0.8;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+  }
```

Comment:
- You have done well so far, but there are some differences between your design and the figma template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- Please ensure that the background is blurred when the navbar is open, consider using the `backdrop-filter` property in css, please fix this 👍 👍 
![Screenshot](https://github.com/sulaimonaa/Portfolio/assets/73607512/3dc9ee32-7ac9-41bb-a9cf-fa530965f8d3)

Figma template:
![Sgwrgawrg](https://github.com/sulaimonaa/Portfolio/assets/73607512/358f0e05-5f63-47fd-a5b6-ceef72422359)

Reply:
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

 - [ ] Nice implementing the design, however, As requested by the previous reviouwer, it would be better if you add a little more bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 



| Your design | Figma design |
| --- | --- |
| ![image](https://github.com/sulaimonaa/Portfolio/assets/79058364/b9fc346c-c685-4b02-ae69-085612977ed3) | ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |

-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.